drivers/i2s/i2s_esp32.c

```diff
@@ -108,12 +112,12 @@ static esp_err_t i2s_esp32_calculate_clock(const struct i2s_config *i2s_cfg, uin
 	uint16_t mclk_multiple = 256;
 
 	if (i2s_cfg == NULL) {
-		LOG_ERR("Input i2s_cfg is NULL");
+		LOG_DBG("Input i2s_cfg is NULL");
```

Comment:
I think you can remove all those logs related to NULL pointers. It already returns INVALID_ARGS.